test/unit/resources/postgres_session_test.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+# author: Aaron Lippold
+
+require 'helper'
+
+describe 'Inspec::Resources::PostgresSession' do
+  it 'verify postgres_session basic init configuration' do
```

Comment:
Trying to do that now, however I can't seem to access the local var 'psql_cmd' correctly.

```
  it 'verify postgres_session create_psql_cmd function' do
    resource = load_resource('postgres_session','myuser','mypass','127.0.0.1')
      _(resource.query("SELECT * FROM STUDENTS;",['testdb'])[psql_cmd]).must_equal "PGPASSWORD='mypass' psql -U myuser -d testdb -h 127.0.0.1 -A -t -c SELECT\\ \\*\\ FROM\\ STUDENTS\\;"
  end
```
Suggestions?